internal/character/serval/trace.go

```diff
@@ -0,0 +1,47 @@
+package serval
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+)
+
+// A4:
+//
+//	At the start of the battle, immediately regenerates 15 Energy.
+//
+// A6:
+//
+//	Upon defeating an enemy, ATK is increased by 20% for 2 turn(s).
+const (
+	A4 = "serval-a4"
+	A6 = "serval-a6"
+)
+
+func initTraces(c *char) {
```

Comment:
Better to at least make it a function of the char rather than its own function
```suggestion
func (c *char) initTraces() {
```
When you change it, remember to change the init in serval.go
c.initTraces()